test/xlsxtream/xml_test.rb

```diff
@@ -42,5 +42,22 @@ def test_escape_value_valid_xml_chars
       expected = safe_value
       assert_equal expected, XML.escape_value(safe_value)
     end
+
+    def test_encode_underscores_using_x005f
+      unsafe_value = "_xDcc2_"
+      safe_value = "_x005f_xDcc2_"
+      assert_equal safe_value, XML.escape_value(unsafe_value)
+    end
+
+    def test_encode_underscores_using_x005f_multiple_occurrences
+      unsafe_value = "_xDcc2_aa_x3d12_bb_xDea3_cc_xDaa5_"
+      safe_value = "_x005f_xDcc2_aa_x005f_x3d12_bb_x005f_xDea3_cc_x005f_xDaa5_"
+      assert_equal safe_value, XML.escape_value(unsafe_value)
+    end
+
+    def test_not_escaping_regular_underscores
+      safe_value = "this_test_does_not_replace_underscores_xDcc2"
+      assert_equal safe_value, XML.escape_value(safe_value)
+    end
```

Comment:
❓ do we want unit tests for the method we added? we are getting the coverage by testing the method that calls our method. Unsure if there are any tests that would be different though

Reply:
Nah I think it's ok given we care about the outcome of `escape_value` and the tests would be identical to your point.